src/test/java/com/jcabi/github/Constants.java

```diff
@@ -0,0 +1,12 @@
+package com.jcabi.github;
+
+/**
+ * Constants for Test classes.
+ * @author Hossein Amerkashi (kkashi01@gmail.com)
+ */
+class Constants {
+    /**
+     * Constant for root repository path.
+     */
+    static final String ROOT_REPO = "/repos";
+}
```

Comment:
@kkashi01 We don't allow global constants in our code base. It's poluting and goes agianst OOP principles.

BTW, is you are curious, we try to work as closely as possible to the "Elegant Objects" methodology: https://www.elegantobjects.org/

"Elegant Objects" is actually a series of books published by the original author of this library.